bin/spark-connect-scala-client.sc

```diff
@@ -0,0 +1,15 @@
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.SparkSession
+
+val conStr = if (sys.env.contains("SPARK_REMOTE")) sys.env("SPARK_REMOTE") else ""
+val sessionBuilder = SparkSession.builder()
+val spark = if (conStr.isEmpty) sessionBuilder.build() else sessionBuilder.remote(conStr).build()
```

Comment:
Do we default to a local connection if none is set?

Reply:
Oh BTW. there does not seem to be SparkSession.Builder.remote() function?

Reply:
Can you also set the user agent to something meaningful?

Reply:
The user agent if not set, defaults to "_SPARK_CONNECT_SCALA". Do you want another name for the script client?